src/registry/GithubRegistry.ts

```diff
@@ -249,7 +252,12 @@ export class GithubRegistry extends BaseRegistry implements IRegistry {
 
   protected async fetch(url: string): Promise<Response> {
     this.logger.debug(`Fetching from github: ${url}`);
-    const response = await fetch(url);
+    const useToken = !(this.proxyUrl && url.startsWith(this.proxyUrl)) && !!this.authToken;
+    const response = await fetch(url, {
```

Comment:
I just note this now and maybe I'm too boomer, but do we need a polyfill for fetch for usage on the frontend?

Reply:
https://caniuse.com/fetch